dao/model.go

```diff
@@ -125,3 +125,12 @@ type BackupRequest struct {
 	SnapshotSpacePercent int
 	Excludes             []string
 }
+
+type BackupEstimate struct {
```

Comment:
Same as other instance - prefer to hold off on move until after Alderaan, due to conflicts with existing calls.